python/ray/util/placement_group.py

```diff
@@ -5,6 +5,25 @@
 import ray
 from ray._raylet import PlacementGroupID, ObjectRef
 
+bundle_reservation_check = None
+
+
+# We need to import this method to use for ready API.
+# But ray.remote is only available in runtime, and
```

Comment:
Isn't ray.remote available after import?

Reply:
Or is this because we are an internal Ray package?

Reply:
Yeah when I just tried defining the method after import, it didn't seem to work. `Or is this because we are an internal Ray package?`= > this is also my guess. Let me actually try if 

```
from ray.worker import remote
@ray.remote...
```

Reply:
Okay confirmed I couldn't find a remote method. I checked it using `dir(ray)`, and it doesn't contain the method for some reasons. Maybe some weird import quirk. Lmk if you know how to fix this!